be/ut/alloc.c

```diff
@@ -575,7 +575,9 @@ M0_INTERNAL void m0_be_ut_alloc_align(void)
 	M0_UT_ASSERT(m0_be_allocator__invariant(a));
```

Comment:
The UT will fail at this point.
Kindly apply the patch shared to add the mutex lock and unlock.

Reply:
Applied patch provided by Yash.